presto-main/src/test/java/com/facebook/presto/cost/TestFilterStatsCalculator.java

```diff
@@ -215,8 +216,8 @@ public void testOrStats()
                                 .distinctValuesCount(20.0)
                                 .nullsFraction(0.0));
 
-        Expression leftExpressionSingleValue = new ComparisonExpression(ComparisonExpressionType.EQUAL, new SymbolReference("x"), new DoubleLiteral("0.0"));
-        Expression rightExpressionSingleValue = new ComparisonExpression(ComparisonExpressionType.EQUAL, new SymbolReference("x"), new DoubleLiteral("-7.5"));
+        Expression leftExpressionSingleValue = new ComparisonExpression(EQUAL, new SymbolReference("x"), new DoubleLiteral("0.0"));
```

Comment:
unrelated change (here & below). Each such a change should be a fixup to the right commit